tests/shipping-preview/models/Delivery Only_Delivery Pickup.model.js

```diff
@@ -0,0 +1,43 @@
+import { setup, visitAndClearCookies } from '../../../utils'
+import {
+  checkShippingPreviewResult,
+  choosePickupShippingPreview,
+  fillShippingPreviewDelivery,
+} from '../../../utils/shipping-actions'
+import { SKUS, ACCOUNT_NAMES } from '../../../utils/constants'
+
+export default function test(account) {
+  describe(`Delivery Only + Delivery/Pickup - ${account}`, () => {
+    before(() => {
+      visitAndClearCookies(account)
+    })
+
+    it('with only pickup', () => {
+      setup({
+        skus: [
+          SKUS.PICKUP_1_SLA_AND_DELIVERY_MULTIPLE_SLA,
+          SKUS.DELIVERY_MULTIPLE_SLA_AND_PICKUP_AT_PORTO_ALEGRE,
+        ],
+        account,
+      })
+
+      const selectors = []
+
+      cy.contains('Calcular').should('be.visible')
+
+      fillShippingPreviewDelivery(account)
+      if (account === ACCOUNT_NAMES.NO_LEAN) {
+        selectors.push({ name: 'PAC', text: 'Em até 10 dias úteis' })
+      } else {
+        selectors.push({ name: 'Mais econômica', text: 'Em até 10 dias úteis' })
+      }
+
+      checkShippingPreviewResult(selectors)
+      choosePickupShippingPreview()
+
+      selectors.push({ text: 'Pronto em até 11 dias' })
```

Comment:
Good question. I took it for granted, but you made me think now. Is the `price` a better alternative?

Reply:
Because the pickup/scheduled delivery selector tend to show only its estimate and price.

Reply:
Doesn't look like a good idea to me. Tests should not rely on a property of a specific product, but rather only on what SLAs it has available. Imagine the pain it would be if we used prices and one day we had to change the items of the test. We would have to update the values of every test! And that's not even what we want to test in the first place...